src/models/program.js

```diff
@@ -22,9 +23,14 @@ export default (sequelize, DataTypes) => {
       allowNull: false,
     },
     programType: DataTypes.STRING,
-    startYear: DataTypes.STRING,

```

Comment:
did you intend to drop the start year column

Reply:
Nope!